pykokkos/lib/util.py

```diff
@@ -8,12 +8,14 @@
 # https://data-apis.org/array-api/2021.12/API_specification/utility_functions.html
 
 def all(x, /, *, axis=None, keepdims=False):
+    return np.all(x)
```

Comment:
`pk.all([1, 1, 1])`  where [1, 1, 1] is pk.View([3], pk.int8)

returns `False` since `[1,1,1] == True` is False. Added a temporary np.all here. Will fix it soon

Reply:
this might not be needed anymore since we support comparing to scalars now